Library/Formula/tiger-vnc.rb

```diff
@@ -0,0 +1,26 @@
+class TigerVnc < Formula
+  homepage "http://tigervnc.org/"
+  url "https://github.com/TigerVNC/tigervnc/archive/v1.4.0.tar.gz"
+  sha1 "36252b10912bcbef44d43c5fa7977ae97988fff3"
+
+  depends_on "cmake" => :build
+  depends_on "gnutls" => :recommended
```

Comment:
I feel a little nervous letting people opt out of this, to be honest, but I guess ` :recommended ` is okay.